### PR TITLE
feat: add first time configs

### LINKED ISSUE #96: Add first time contributor message
# Specification

The two business problems to solve are: 

1. a custom message that partners can use to greet new hunters
2. show a list of all of the bot commands that the new hunter can invoke.

We must create new config properties: 
- [ ] `FIRST_TIME_CONTRIBUTOR_GREETING_HEADER`: string;
- [ ] `FIRST_TIME_CONTRIBUTOR_GREETING_HELP_MENU`: boolean;
- [ ] `FIRST_TIME_CONTRIBUTOR_GREETING_FOOTER`: string;

Post the header first, then the help menu, and lastly the footer in a single "first time contributor greeting" comment.

- [ ] Be sure to also update the README.md with these config properties.

# Example Input

```
FIRST_TIME_CONTRIBUTOR_GREETING_HEADER="Thank you for contributing to UbiquiBot! Please be sure to set your wallet address before completing your first bounty so that the automatic payout upon task completion will work for you."
FIRST_TIME_CONTRIBUTOR_GREETING_HELP_MENU=true
FIRST_TIME_CONTRIBUTOR_GREETING_FOOTER="###### Also please star this repository and [@ubiquity/devpool-directory](https://github.com/ubiquity/devpool-directory/) to show your support. It helps a lot!"
```

# Example Output

Thank you for contributing to UbiquiBot! Please be sure to set your wallet address before completing your first bounty so that the automatic payout upon task completion will work for you. 

```
- /assign: Assign the origin sender to the issue automatically.
- /unassign: Unassign the origin sender from the issue automatically.
- /help: List all available commands.
- /payout: Disable automatic payment for the issue.
- /multiplier: Set bounty multiplier (for treasury)
- /allow: Set access control. (Admin Only)
- /wallet: <WALLET_ADDRESS | ENS_NAME>: Register the hunter's wallet address. 
  ex1: /wallet 0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266 
  ex2: /wallet vitalik.eth
```

### PR DESCRIPTION
**Pull request by EtherealGlow**
Resolves #96

### CODE REVIEW

**Comment by EtherealGlow** [comment]
some name might be a little different

File: src/handlers/comment/handlers/first.ts [hunk 1]
```diff
@@ -25,7 +29,15 @@ export const verifyFirstCheck = async (): Promise<void> => {
       const isFirstComment = resp.data.filter((item) => item.user?.login === payload.sender.login).length === 1;
       if (isFirstComment) {
         //first_comment
-        const msg = `${generateHelpMenu()}\n@${payload.sender.login}`;
+        if (firstTimeContributorHeader) {
+          msg += `${firstTimeContributorHeader}\n`;
+        }
+        if (firstTimeContributorHelpMenu) {
+          msg += `${generateHelpMenu()}\n@${payload.sender.login}\n`;
```
Comment (whilefoo) [inline src/handlers/comment/handlers/first.ts:36]: If I read the issue comments correctly, we don't want to tag the user? @pavlovcik

**Review by whilefoo — CHANGES_REQUESTED** [review]
Code looks good.

After we finish discussing the structure of the config, please update the `README.md` and `ubiquibot-config-default.json` (by default the feature should be disabled).

File: .github/ubiquibot-config.yml [hunk 2]
```diff
@@ -36,3 +36,6 @@ max-concurrent-bounties: 2
 promotion-comment: "\n<h6>If you enjoy the DevPool experience, please follow <a href='https://github.com/ubiquity'>Ubiquity on GitHub</a> and star <a href='https://github.com/ubiquity/devpool-directory'>this repo</a> to show your support. It helps a lot!</h6>"
 register-wallet-with-verification: false
 assistive-pricing: true
+first-time-contributor-header: Thank you for contributing to UbiquiBot! Please be sure to set your wallet address before completing your first bounty so that the automatic payout upon task completion will work for you."
+first-time-contributor-help-menu: true
+first-time-contributor-footer: "###### Also please star this repository and [@ubiquity/devpool-directory](https://github.com/ubiquity/devpool-directory/) to show your support. It helps a lot!"
```
[the remark this answers is not in the document]
Comment (pavlovcik) [reply .github/ubiquibot-config.yml:41]: Agreed with all of that!

File: src/handlers/comment/handlers/first.ts (hunk 1 above)
Comment (pavlovcik) [reply src/handlers/comment/handlers/first.ts:36]: I think tagging the user is nice to get their attention for this. Just make sure it's in normal text, and not monospace or else the tag doesn't work.

**Pull request closed without merging**